core/contracts/financial_templates/PricelessPositionManager.sol

```diff
@@ -294,9 +376,22 @@ contract PricelessPositionManager is Testable {
         delete positions[sponsor];
     }
 
-    function _getPositionData(address sponsor) internal view returns (PositionData storage position) {
-        position = positions[sponsor];
-        require(position.isValid, "Position does not exist");
+    function _getPositionData(address sponsor)
+        internal
+        view
+        onlyCollateralizedPosition(sponsor)
+        returns (PositionData storage)
+    {
+        return positions[sponsor];
+    }
+
+    function _getIdentifierWhitelistAddress() internal view returns (address) {
+        bytes32 identifierWhitelistInterface = "IdentifierWhitelist";
+        return finder.getImplementationAddress(identifierWhitelistInterface);
+    }
```

Comment:
Surprised the linter isn't complaining about needing an empty line here? Or is this just the github diff being weird?

Reply:
actually it was missing a blank line. strange that the linter didnt catch it.